pulpcore/app/serializers/purge.py

```diff
@@ -0,0 +1,19 @@
+from datetime import datetime, timedelta, timezone
+from gettext import gettext as _
+
+from rest_framework import serializers
+
+from pulpcore.app.serializers import ValidateFieldsMixin  # noqa
+from pulpcore.constants import TASK_FINAL_STATES
+
+
+class PurgeSerializer(serializers.Serializer, ValidateFieldsMixin):
+    finished_before = serializers.DateTimeField(
+        help_text=_("Earliest finished-timestamp to **NOT** purge. Format '%Y-%m-%d[T%H:%M:%S]'"),
```

Comment:
I alread LGTM'd this but could this be stated simpler:
```suggestion
        help_text=_("Purge tasks completed earlier than this timestamp. Format '%Y-%m-%d[T%H:%M:%S]'"),
```

Reply:
:+1:

Reply:
I you push lmk and we can merge.